### DISCUSSION TITLE
How to add multiple characters through CLI?

### DISCUSSION THREAD

**Discussion by SleepingPanda** [comictagger/comictagger#408] [Q&A] [answered]
I think I might be an idiot for not figuring this out on my own. I'll admit i'm not the sharpest tool in the shed.

I've tried:
`comictagger -m "characters=Sana Kashimura,Zoroku Kashimura"`
Which returns:
`comictagger: error: argument -m/--metadata: invalid parse_metadata_from_string value: 'characters=Sana Kashimura,Zoroku Kashimura'`

Next tried:
`comictagger -m "characters=Sana Kashimura,characters=Zoroku Kashimura"`
That only saves the last tag 'Zoroku Kashimura' instead of both.

I then tried:
comictagger -m "characters='Sana Kashimura','Zoroku Kashimura'"
Returns the first error:
`comictagger: error: argument -m/--metadata: invalid parse_metadata_from_string value: "characters='Sana Kashimura','Zoroku Kashimura'"`

Can I have some help with this?

**Comment by lordwelch** [comment, marked as answer]
Your just missing the `^` sign to escape the comma, you type it via `Shift+6`.
so you would put `comictagger -m "characters=Sana Kashimura^,Zoroku Kashimura"`

**Reply by SleepingPanda** [reply]
I didn't think the comma needed to be escaped. Wow. I knew it was something simple but this still makes me feel like a moron. Thanks for your answer!
